backend/fms_core/import_tool/handlers/experiment_run.py

```diff
@@ -0,0 +1,67 @@
+from datetime import datetime
+from pandas import pandas as pd
+
+from fms_core.models import ExperimentType, PropertyType, Instrument, Container, Sample
+from ._generic import GenericHandler
+from ...services import create_experiment_run
+
+
+
+
+class ExperimentRunHandler(GenericHandler):
+    def __init__(self, experiment_type_obj, instrument, container, start_date,
+                 sample_rows, properties, protocols_dict, properties_by_name_dict):
+
+        self.experiment_run = {'experiment_type': experiment_type_obj, 'instrument': None, 'container': None,
```

Comment:
Does this have to be a class attribute ? as far as i see this is local storage. Not sure why you even copy all of this locally...

Reply:
Yeah, for now it doesnt have to be a class attribute. I thought of generating the attributes automatically in the Handler objects at some point (and the idea is not totally dropped; I wanted to try with other templates to see how it would turn out), but for now it is not necessary.